docs/en/thirdparty/wokwi.rst

```diff
@@ -0,0 +1,42 @@
+#####
+Wokwi
+#####
+
+.. warning::
+  Note that this tool is **not maintained by the ESP32 Arduino Core team**, so we cannot provide support or guarantee that it will work as expected.
```

Comment:
```suggestion
  This tool is **not maintained by the ESP32 Arduino Core team**, so we cannot provide support or guarantee that it will work as expected.
```